### DISCUSSION TITLE
Whatsapp api

### DISCUSSION THREAD

**Discussion by tuyewxx** [aldinokemal/go-whatsapp-web-multidevice#117] [Q&A] [answered]
Bang untuk whatsapp http api harusnya udah bisa ya saya build lewat docker. 
dengan metode seperti apa saya menggunakan pats ip:port/send/message?phone=[number]&text=[text]
itu engga bisa bang harus nya seperti apa?

**Comment by aldinokemal** [comment]
metodenya POST bro, jadinya yang dikirim body json bukan query params

**Reply by tuyewxx** [reply]
harusnya gimana bang?
sya butuh server url

**Reply by tuyewxx** [reply]
![image](https://github.com/aldinokemal/go-whatsapp-web-multidevice/assets/86847407/cf9e7af1-89bb-4e9f-8d89-b00a33ef1c44)

**Reply by aldinokemal** [reply]
coba pake [Postman](https://www.postman.com/downloads/) aja biar bisa simulate metode POST

kalau pake browser mostly cuma bisa GET

**Reply by aldinokemal** [reply]
Kalau awal bisa dicoba running aplikasinya dulu
uda ada UI nya untuk bantu login dan coba submit message

**Reply by aldinokemal** [reply, marked as answer]
To see an example of an API, see this document.
https://bump.sh/aldinokemal/doc/go-whatsapp-web-multidevice

**Reply by isandev-20** [reply]
> coba pake [Postman](https://www.postman.com/downloads/) aja biar bisa simulate metode POST
> 
> kalau pake browser mostly cuma bisa GET

Apa bisa dengan methode get bang?
